service/lib/ops/policyOps.js

```diff
@@ -39,16 +39,26 @@ function insertPolicies (client, policies, cb) {
   client.query(sql, cb)
 }
 
-function deletePolicies (client, ids, cb) {
-  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deletePolicies (client, ids, orgId, cb) {
+  client.query(SQL`DELETE FROM policies WHERE id = ANY (${ids}) AND org_id=${orgId}`, utils.boomErrorWrapper(cb))
 }
 
-function deleteTeamsAssociations (client, ids, cb) {
-  client.query(SQL`DELETE FROM team_policies WHERE policy_id = ANY (${ids})`, utils.boomErrorWrapper(cb))
+function deleteTeamsAssociations (client, ids, orgId, cb) {
+  client.query(SQL`
+               DELETE FROM team_policies AS p
+               USING teams AS t
+               WHERE t.id = p.team_id
+                 AND p.policy_id = ANY (${ids})
+                 AND t.org_id = ${orgId}`, utils.boomErrorWrapper(cb))
```

Comment:
That's a good question. 

This is how I'm used to indent SQL. The reason is that `AND` keywords children to the `WHERE` clause. It really helps when queries become more complex.

Also putting the SQL statements into the javascript code might become problematic when the queries become more complex. It's harder to test directly with psql and harder to read. Currently most queries are very straightforward but it's mostly because we make a bunch of them for each user action.